lib/active_shipping/carriers/usps_returns.rb

```diff
@@ -0,0 +1,99 @@
+require 'cgi'
```

Comment:
If you're using CGI, can you require this at a higher level? In `lib/active_shipping.rb` or something. Be explicit about what is using it and what is being required.